apps/picsa-tools/budget-tool/src/app/components/budget-summary/budget-summary.component.scss

```diff
@@ -0,0 +1,29 @@
+.summary-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-bottom: 1.2rem;
```

Comment:
nit(non-blocking)
It would probably be better if the component doesn't define its own margin, and instead allows the parent to do so.
To achieve this you should add a css style for:
```css
host:{
   display: block
}
```
then when the component is used it should respond as expected to styling, e.g.
```html
<budget-summary style=margin-bottom:1.2rem />
```